src/applications/personalization/dashboard-2/components/health-care/Appointments.jsx

```diff
@@ -21,7 +21,7 @@ export const Appointments = ({ appointments }) => {
   }
 
   return (
-    <div className="vads-u-display--flex vads-u-flex-direction--column large-screen:vads-u-flex--1 large-screen:vads-u-margin-right--3 vads-u-margin-bottom--2p5">
```

Comment:
This extra right margin was creating a visual bug

![Uploading Screen Shot 2021-04-12 at 11.39.45 AM.png…]()